<Xcpt>
  <Time>28.04.2017 00:08:25</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:09:56</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:11:16</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:11:47</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:12:39</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:13:31</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:14:00</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>28.04.2017 00:14:52</Time>
  <Info>id: 8</Info>
</Info>
<Info>
  <Time>28.04.2017 00:14:52</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>28.04.2017 00:14:53</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>28.04.2017 00:14:53</Time>
  <Info>Card: 383</Info>
</Info>
<Info>
  <Time>28.04.2017 00:14:53</Time>
  <Info>Card: 350</Info>
</Info>
<Info>
  <Time>28.04.2017 00:14:53</Time>
  <Info>Card: 687</Info>
</Info>
<Info>
  <Time>28.04.2017 00:14:53</Time>
  <Info>Card: 564</Info>
</Info>
<Info>
  <Time>28.04.2017 00:14:53</Time>
  <Info>Card: 612</Info>
</Info>
<Xcpt>
  <Time>28.04.2017 00:14:54</Time>
  <Exception>
    <Source>System</Source>
    <Message>Fehler beim Senden von Mail.</Message>
    <Stack>   bei System.Net.Mail.SmtpClient.Send(MailMessage message)
   bei CardGame.Web.HtmlHelpers.EmailHelper.SendEmail(String emailaddress, String emailsubject, String text) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.Web\HtmlHelpers\EmailHelper.cs:Zeile 38.</Stack>
    <InnerException>
      <Source>System</Source>
      <Message>Der Remotename konnte nicht aufgelöst werden: 'srv08.itccn.loc'</Message>
      <Stack>   bei System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress&amp; address, Socket&amp; abortSocket, Socket&amp; abortSocket6)
   bei System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   bei System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   bei System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   bei System.Net.Mail.SmtpClient.GetConnection()
   bei System.Net.Mail.SmtpClient.Send(MailMessage message)</Stack>
    </InnerException>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:18:54</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:26:15</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:27:11</Time>
  <Exception>
    <Source>EntityFramework</Source>
    <Message>The operation failed: The relationship could not be changed because one or more of the foreign-key properties is non-nullable. When a change is made to a relationship, the related foreign-key property is set to a null value. If the foreign-key does not support null values, a new relationship must be defined, the foreign-key property must be assigned another non-null value, or the unrelated object must be deleted.</Message>
    <Stack>   bei System.Data.Entity.Core.Objects.ObjectContext.PrepareToSaveChanges(SaveOptions options)
   bei System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   bei System.Data.Entity.Internal.InternalContext.SaveChanges()
   bei CardGame.DAL.Logic.DeckManager.UpdateDeckById(Int32 id, List`1 deckCards) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\DeckManager.cs:Zeile 103.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>28.04.2017 00:29:03</Time>
  <Exception>
    <Source>EntityFramework</Source>
    <Message>The operation failed: The relationship could not be changed because one or more of the foreign-key properties is non-nullable. When a change is made to a relationship, the related foreign-key property is set to a null value. If the foreign-key does not support null values, a new relationship must be defined, the foreign-key property must be assigned another non-null value, or the unrelated object must be deleted.</Message>
    <Stack>   bei System.Data.Entity.Core.Objects.ObjectContext.PrepareToSaveChanges(SaveOptions options)
   bei System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   bei System.Data.Entity.Internal.InternalContext.SaveChanges()
   bei CardGame.DAL.Logic.DeckManager.UpdateDeckById(Int32 id, List`1 deckCards) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame.DAL\Logic\DeckManager.cs:Zeile 103.</Stack>
  </Exception>
</Xcpt>
